src/charm.py

```diff
@@ -84,14 +104,15 @@ def __init__(self, *args):
                 "logging": "logging",
                 "metrics": "metrics-endpoint",
                 "s3": "s3",
-                "tracing": "self-tracing",
+                "charm-tracing": "self-charm-tracing",
+                "workload-tracing": "self-workload-tracing",
             },
             nginx_config=NginxConfig(server_name=self.hostname).config,
             workers_config=self.tempo.config,
-            tracing_receivers=self.requested_receivers_urls,
             resources_requests=self.get_resources_requests,
             container_name="charm",
             remote_write_endpoints=self.remote_write_endpoints,  # type: ignore
+            workload_tracing_protocols=["jaeger_thrift_http"],
```

Comment:
let's add a comment that this will "soon" be replaced by OTEL as soon as they release from main.